Answers.md

```diff
@@ -1,3 +1,18 @@
-### Your responses to the short answer questions should be laid out here using markdown.
 
-For help with markdown syntax, [go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+# Assessment Answers
+ ### 1. Describe the differences between a Docker container and a virtual machine. What makes a container more aptly-suited for portability?
+    Docker containers use a lot less resources and generally run faster than VMs. Dockers also don't have system overhead to worry about.
+
+ ### 2. Given the commend `docker run -p 49160:8080 -d <your_docker_username>/<your_docker_image_name>`, what does `49160:8080` specify?
+    `49160:8080` is the port number used for the application.
+
+ ### 3. What is the main purpose of using a "container orchestration platform" such as Kubernetes or Docker Swarm?
```

Comment:
Container Orchestration Platforms are systems that allow you to more easily provision your docker images in a large system. Platforms like Kubernetes or Docker Swarm have tools that make it easier to spin up, move, and dedicate system resources to each container.